src/services/api.tsx

```diff
@@ -78,10 +78,17 @@ const api = {
         return allFavorites;
     },
 
-    async uploadAvatar(image: any) {
+    async uploadAvatar(image: any):Promise<string> {
         const imageRef = storageRef.child(image.name);
-        await imageRef.put(image)
+        await imageRef.put(image);
         return await imageRef.getDownloadURL();
+    },
+
+    async deleteImageFromStorage(fullpath: string):Promise<void> {
+        let splitedPath = fullpath.split('/');
```

Comment:
const splitedPath
const imgName
const desertRef

as you don't change it

Reply:
okay, thanks for the advice!